config/locales/en.yml

```diff
@@ -108,6 +108,7 @@ en:
     questions:
       still_working:
         title: "Are you still going in to work even though you’re not a key worker?"
+        title_caption: "Going in to work"
```

Comment:
I wonder whether the structure of the YAML should be re-jigged a bit because quite a few questions are going to need this caption?

Perhaps:
```
coronavirus_form:
  groups:
      going_to_work:
         title: Going in to work
         questions:
             still_working:  
                  title: Are you still going in to work even though you’re not a key worker?
...
```

Then you'd be able to reference the same group across different views:
`t("coronavirus_form.groups.going_to_work.title")`

But the rest of the references would need to be expanded to include `groups.going_to_work`

What do you think?

Reply:
I think this is a good structure for the question that have subtitles. There are three questions without subtitles we'd need to invent groups for - not a big deal, but another thing to consider.

Currently only one of the groups has three questions in it; three groups have two questions, and the remaining six groups have a single question. You'll end up with one subtitle repeated three times, and another subtitle repeated twice. I don't think this is duplication is a problem in a data structure that people are already familiar with.

I think that the structure is neat and a good idea, but think for this case this is premature optimisation. But I do think we should revisit this later if the i18n YAML becomes too repetitive.

Reply:
Assuming that the design of the results page doesn't change, know which group a question belongs to will useful for that too.

Reply:
Makes sense to me!

Reply:
I think we were properly modelling each question, we could get away with having a “group” property on the question. Something for @huwd and I to think about.